docs/conf.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
```

Comment:
Awkward wording on line 5, but no big deal (up to you if you think its worth changing)

Reply:
It's an autogenerated file so I think I'll keep it the way it is.